templates/default/slurm/parallelcluster_slurm_cloudbursting.conf.erb

```diff
@@ -0,0 +1,9 @@
+[slurm_resume]
```

Comment:
minor: at this point since we are duplicating the config anyway why not using 2 different files?